platforms/core-configuration/file-collections/src/integTest/groovy/org/gradle/api/file/RelativePathFilesIntegrationTest.groovy

```diff
@@ -97,7 +97,7 @@ class RelativePathFilesIntegrationTest extends AbstractIntegrationSpec {
 
     // TODO: write a similar test for the RegularFileProperty
     @ToBeFixedForConfigurationCache
```

Comment:
🧪 I missed the annotation in the original PR. So it seems the underlying hasn't been fixed or at least not in a way that works with this test?